src/common/database.c

```diff
@@ -2366,6 +2366,51 @@ static int _upgrade_library_schema_step(dt_database_t *db, int version)
 
     new_version = 39;
   }
+  else if(version == 39)
+  {
```

Comment:
Not sure to know how it works exactly, but considering most people will do an upgrade from a version < 38, wouldn't it be better to do the upgrade from 38 and 39 in one time ? In order to avoid unneeded table creation-deletion-recreation ?

Reply:
Just tested, and apart from that, everything is fine on my side.
Thanks a lot for this quick change ;)

Reply:
Except that my previous change has been merged and so I need an incremental upgrade otherwise I'll break current DB. And this will happen to me BTW :)